### PR TITLE
Create student user at end of new signup flow

### PR DESCRIPTION
**Pull request by TurnerRiley**
Creates student user entry (and associated parent email preference entry if applicable) at the end of the new signup flow.

## New student fills in all fields with a parent opting in to receiving emails
studentallfields

User entry:
studentallfields db entry

Parent `EmailPreference` entry:
studentallfields parent email pref

## New student with a parent opting out of receiving emails
studentparentoptout

User entry:
studentparentoptout db

Parent `EmailPreference` entry:
studentparentoptout parent email entry

## New student without a parent filling in only required fields (name, age, state)
minfields

User entry:
minfields db

## Links
Jira ticket: [here](https://codedotorg.atlassian.net/jira/software/c/projects/ACQ/boards/64?assignee=60d62161f65054006980bd71&selectedIssue=ACQ-2385)

## Testing story
Local testing and adding unit tests.

### CODE REVIEW

File: apps/src/signUpFlow/FinishStudentAccount.tsx [hunk 1]
```diff
@@ -13,17 +13,14 @@ import {
...
 
 import locale from './locale';
-import {
-  IS_PARENT_SESSION_KEY,
-  PARENT_EMAIL_SESSION_KEY,
-  PARENT_EMAIL_OPT_IN_SESSION_KEY,
-  USER_AGE_SESSION_KEY,
-  USER_STATE_SESSION_KEY,
-  USER_GENDER_SESSION_KEY,
-} from './signUpFlowConstants';
```
Comment (TurnerRiley) [inline apps/src/signUpFlow/FinishStudentAccount.tsx:26]: All of these fields are set on this page so it didn't really make sense to use `sessionStorage` for them

**Review by hannahbergam — COMMENTED** [review]
This looks right to me! And we still have a bunch of front end validation to ensure those fields don't get sent blank in the fetch right?

**Comment by TurnerRiley** [comment]
> This looks right to me! And we still have a bunch of front end validation to ensure those fields don't get sent blank in the fetch right?

Yep the button isn't enabled until the required fields are filled in!

**Review by hannahbergam — APPROVED** [review]
This looks right to me! Thanks for these tests!

**Pull request merged**